compiler/optimizer/Inliner.cpp

```diff
@@ -376,6 +376,11 @@ TR_InlinerBase::setInlineThresholds(TR::ResolvedMethodSymbol *callerSymbol)
          _nodeCountThreshold *= 2;
       }
 
+   static const char *g = feGetEnv("TR_NumCallSites");
```

Comment:
Why isn't the environment variable called `TR_MaxInliningCallSites` to match the field that it sets?

Reply:
Addressed, thanks